spec/features/browsing_authors_spec.rb

```diff
@@ -23,11 +26,24 @@
         end
 
         scenario "I should see the author's detailed information" do
-          within '.author .name' do
+          within '.author-name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.